tests/src/Functional/Analytics/PostsTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Lullabot\Parsely\Tests\Functional\Analytics;
+
+use Lullabot\Parsely\Analytics\Posts;
+use Lullabot\Parsely\Post;
+use Lullabot\Parsely\PostList;
+use Lullabot\Parsely\Tests\Functional\FunctionalTestBase;
+
+class PostsTest extends FunctionalTestBase
+{
+
+    /**
+     * A test that loads 10 posts.
+     */
+    public function testExecute()
```

Comment:
This test is helpful for debugging the API. Follow the steps in `phpunit.xml.dist` and run this test in PHPUnit or at the CLI with `vendor/bin/phpunit --testsuite functional` and you'll get something like this:

<img width="1001" alt="poststest php parse ly-php 2018-09-14 14-45-43" src="https://user-images.githubusercontent.com/255023/45569312-2777c580-b82d-11e8-903b-4a0b69344730.png">

Reply:
Hm show this again?